rustls/src/msgs/handshake.rs

```diff
@@ -1,4 +1,5 @@
 #![allow(non_camel_case_types)]
+use crate::dns_name;
```

Comment:
I feel like we should just import `DnsName` and `DnsNameRef` directly -- it doesn't seem like there's much potential for confusion with another type, and it seems more in line with the way we usually do imports.